arm_robots/launch/med.launch

```diff
@@ -0,0 +1,66 @@
+<launch>
+<!--    <env name="ROSCONSOLE_CONFIG_FILE"-->
+<!--         value="$(find arm_robots)/config/victor_rosconsole.conf"/>-->
+
+    <arg name="sim" default="false"/>
+    <arg name="gui" default="true"/>
+    <arg name="update_description" default="true"/>
+    <arg name="verbose" default="false"/>
+
+    <group if="$(arg update_description)">
+        <!-- uploads robot description and setups up other parameters -->
+        <include file="$(find iiwa_med_moveit_config)/launch/planning_context.launch">
+            <arg name="load_robot_description" value="true"/>
+        </include>
+    </group>
+
+
+    <group if="$(arg sim)">
+        <!-- Start gazebo -->
+        <group> <!-- this empty group tag creates a scope for the "remap" command -->
+            <!-- because we use ros_control only for the joints of the arms, not the grippers,
+                 this is used to rename the joint_states topic it published -->
+<!--            <remap from="med/joint_states" to="med/arms_joint_states"/>-->
+            <include file="$(find gazebo_ros)/launch/empty_world.launch">
+                <arg name="gui" value="$(arg gui)"/>
+                <arg name="paused" value="true"/>
+                <arg name="verbose" value="$(arg verbose)"/>
+            </include>
+        </group>
+
+        <!-- Spawn the robot in gazebo -->
+        <node name="urdf_spawner" pkg="gazebo_ros" type="spawn_model" respawn="false" output="screen"
+              args="-urdf -model med -param robot_description"/>
+
+        <!-- start the ROS controllers, only used in gazebo. This merges gripper and arm joint states -->
+        <include file="$(find victor_control)/launch/med_control_gazebo.launch">
```

Comment:
why is this under victor_control ?

Reply:
This is because for now I've been throwing the med configs into the kuka_iiwa_interface existing packages. On my todo list today is to pull all these med configs/launches out into their own packages that rely on your kuka_iiwa_interface packages.